app/src/main/res/layout/patientsummary_bpplaceholderitem_content.xml

```diff
@@ -4,14 +4,13 @@
   android:id="@+id/placeHolderMessageTextView"
   android:layout_width="match_parent"
```

Comment:
In addition, I think after this PR, the `PatientSummaryConfig#numberOfBpPlaceholders` property will also be unused.

Reply:
For now, it's being used by `BloodSugarSummaryView`, once I completed the new `BloodSugarSummaryView` I will remove that property as well